packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason.jsx

```diff
@@ -140,7 +140,7 @@ class DeactivateAccountReason extends React.Component {
                 this.setState({ remaining_characters: 0 });
             }
 
-            if (!/^[0-9A-z .,'-]*$/.test(final_value)) {
+            if (!/^[ a-zA-Z0-9.,'"-\s]*$/.test(final_value)) {
```

Comment:
why are we adding double quotes? It is not mentioned in the error message below.
```suggestion
            if (!/^[a-zA-Z0-9.,'-\s]*$/.test(final_value)) {
```

Reply:
Thanks, Balakrishna. I removed double quotes from this regular expression